packages/ec2-metadata-service/src/MetadataServiceOptions.ts

```diff
@@ -34,5 +34,5 @@ export interface MetadataServiceOptions {
   /**
    * the number of seconds to sleep in-between retries and/or a customer provided backoff function to call.
```

Comment:
docs should say:

> If an async function is used, it will be awaited and its return value ignored.
> If a function returning a number if used, the returned number will be used as a millisecond duration to wait before the following retry attempt.